lib/solid.ts

```diff
@@ -87,7 +162,7 @@ export async function getCurrentPod(): Promise<Pod> {
 }
 
 // TODO: don't use deprecated type
```

Comment:
welk type hier is deprecated

Reply:
die JsonLd. Is iets flexibeler dan AccessGrant type

Reply:
Zet je dat erbij in die comment?